recipes/perl-atlas-modules/meta.yaml

```
@@ -38,7 +40,7 @@ requirements:
    - perl-datetime-format-strptime
    - perl-dbd-mysql
    - perl-dbd-pg
-   - perl-dbi<1.642
+   - perl-dbi <1.642
```

perl-dbi was migrated to conda-forge, but only version 1.643 is available there.
If you really need to pin an old version, it needs to come from bioconda, I think.

```suggestion
   - bioconda::perl-dbi <1.642
```

---

It looks like it didn't make a difference, still failing with:
```
package perl-dbi-1.641-pl526_2 is excluded by strict repo priority
```

---

I think this is failing ` package perl-xml-simple-2.22-0 requires perl-xml-sax-expat, but none of the providers can be installed`